Index: sites/all/modules/external/facebook_comments_box/facebook_comments_box.module
===================================================================
--- sites/all/modules/external/facebook_comments_box/facebook_comments_box.module	(revision 63210)
+++ sites/all/modules/external/facebook_comments_box/facebook_comments_box.module	(working copy)
@@ -92,8 +92,8 @@
 
         // Set the content of the block.
         $fcb_content = '<div class="facebook-comments-box">';
-        $fcb_content .= '<div id="fb-root"></div>';
-        $fcb_content .= '<script src="http://connect.facebook.net/en_US/all.js#xfbml=1"></script>';
+        //$fcb_content .= '<div id="fb-root"></div>';
+        //$fcb_content .= '<script src="http://connect.facebook.net/en_US/all.js#xfbml=1"></script>';
         $fcb_content .= '<fb:comments href="' . $fcb_url . '" ';
         $fcb_content .= ' num_posts="' . check_plain(variable_get('facebook_comments_box_default_comments', 10)) . '" ';
         $fcb_content .= ' width="' . check_plain(variable_get('facebook_comments_box_default_width', 400)) . '" ';
