diff --git a/workbench_moderation.module b/workbench_moderation.module
index b652b6f..66eb985 100644
--- a/workbench_moderation.module
+++ b/workbench_moderation.module
@@ -1193,12 +1193,7 @@ function workbench_moderation_node_current_load($node) {
 function workbench_moderation_node_live_load($node) {
   // Is there a live revision of this node?
   if (isset($node->workbench_moderation['published']->vid)) {
-    // If the live revision is not this revision, we need to load that revision
-    if ($node->vid != $node->workbench_moderation['published']->vid) {
-      return node_load($node->nid, $node->workbench_moderation['published']->vid, TRUE);
-    }
-    // This is the live node, return it
-    return $node;
+    return node_load($node->nid, $node->workbench_moderation['published']->vid, TRUE);
   }
 }
 
