Index: sites/all/modules/external/media_browser_plus/media_browser_plus.module
===================================================================
--- sites/all/modules/external/media_browser_plus/media_browser_plus.module	(revision 66716)
+++ sites/all/modules/external/media_browser_plus/media_browser_plus.module	(working copy)
@@ -610,6 +610,9 @@
  * Returns a select form item with a selectable media folders.
  */
 function _media_browser_plus_folder_form() {
+  $path = drupal_get_path('module', 'media_browser_plus');
+  drupal_add_css($path.'/css/media_browser_plus.admin.css');
+  
   $list = _media_browser_plus_folder_list();
   return array(
     '#type' => 'select',
@@ -2032,7 +2035,7 @@
     $path .= $root_folder . '/';
   }
   $root_folder_term = media_browser_plus_get_media_root_folder();
-  if ($term) {
+  /*if ($term) {
     $parents = array_reverse(taxonomy_get_parents_all($term->tid));
     if (is_array($parents) && !empty($parents)) {
       foreach ($parents as $parent) {
@@ -2043,7 +2046,7 @@
         }
       }
     }
-  }
+  }*/
   return rtrim($path, '/');
 }
 
@@ -2088,6 +2091,12 @@
  *   Replace behavior when the destination file already exists.
  */
 function media_browser_plus_move_file($tid, $media, $replace = FILE_EXISTS_RENAME) {
+  switch ($media->filemime) {
+    case 'video/youtube':
+      return;
+      break;
+  }
+  
   // Don't change the uri for media files with external source.
   if (strpos($media->uri, 'public') === FALSE && strpos($media->uri, 'private') === FALSE) {
     file_save($media);
@@ -2103,7 +2112,7 @@
     $folder = media_browser_plus_folder_load($tid);
     $path = media_browser_plus_construct_dir_path($folder);
     file_prepare_directory($path, FILE_CREATE_DIRECTORY);
-    file_move($media, $path, $replace);
+    //file_move($media, $path, $replace);
   }
 }
 
@@ -2192,9 +2201,9 @@
         }
         // Also delete the folder when it's empty.
         if ($all_files_deleted) {
-          if (!@drupal_rmdir($folder_path)) {
+          /*if (!@drupal_rmdir($folder_path)) {
             drupal_set_message(t('Unable to delete the folder (!path) on the disk', array('!path' => $folder_path)), 'error');
-          }
+          }*/
         }
       }
     }
