diff --git a/includes/bootstrap.inc b/includes/bootstrap.inc
index 2123a80..52fcb67 100644
--- a/includes/bootstrap.inc
+++ b/includes/bootstrap.inc
@@ -1048,7 +1048,7 @@ function drupal_page_get_cache($check_only = FALSE) {
     if ($da_site) {
       $root = $base_root . '/' . $da_site;
     }
-    $cache = cache_get($base_root . request_uri(), 'cache_page');
+    $cache = cache_get($root . request_uri(), 'cache_page');
     if ($cache !== FALSE) {
       $cache_hit = TRUE;
     }
diff --git a/includes/common.inc b/includes/common.inc
index 42dab65..535be02 100644
--- a/includes/common.inc
+++ b/includes/common.inc
@@ -5221,7 +5221,7 @@ function drupal_page_set_cache() {
     $page_compressed = variable_get('page_compression', TRUE) && extension_loaded('zlib');
 
     $cache = (object) array(
-      'cid' => $base_root . request_uri(),
+      'cid' => $root . request_uri(),
       'data' => array(
         'path' => $_GET['q'],
         'body' => ob_get_clean(),
